Mlem/ContentView.swift

```diff
@@ -37,13 +37,19 @@ struct ContentView: View {
                     Label(computeUsername(account: currentActiveAccount), systemImage: "person.circle")
                         .environment(\.symbolVariants, tabSelection == 3 ? .fill : .none)
                 }.tag(3)
+                
+                NavigationView {
+                    SearchView(account: currentActiveAccount)
+                } .tabItem {
+                    Label("Search", systemImage: "text.magnifyingglass")
```

Comment:
Can we make this `magnifyingglass` when it's not selected and `text.magnifyingglass` when it is?